src/functionals/SCANc.cpp

```diff
@@ -0,0 +1,25 @@
+#include "functional.hpp"
+#include "constants.hpp"
+#include "SCAN_like_eps.hpp"
+
+template<class num>
+static num SCANC(const densvars<num> &d)
+{
+  num eps_c = SCAN_eps::SCAN_C(d, 0, 0, 0); 
+ 
+  return eps_c;
+}
+
+FUNCTIONAL(XC_SCANC) = {
+  "SCAN correlation functional",
+  "SCAN correlation functional.\n"
+  "J. Sun, A. Ruzsinszky, and J. P. Perdew, Phys. Rev. Lett. 115, 036402 (2015)."
+  "Implemented by James Furness\n",
```

Comment:
You could consider adding your GitHub handle here (and similarly for the other new functionals)

Reply:
What's the best format for that? Simply "Implemented by James Furness (github: JFurness1)"?

Reply:
I think `@JFurness1` would be standard _de facto_